python/torch_mlir/dialects/torch/importer/jit_ir/csrc/import_options.h

```diff
@@ -33,6 +33,10 @@ struct ImportOptions {
   // In that case, the appropriate shape information is provided via the type
   // bound annotations on the function arguments instead.
   bool ignoreExistingTensorShapesAndDtypes = false;
+
+  // Whether to add the FXOutputName attribute from the debug name of the jit
+  // node.
+  bool addFxOutputName = false;
```

Comment:
In the context of torch-mlir this output name would be the tensor "SSA" name we see in the FX graph right? Maybe its worth mentioning it in the comment.

We have just been using OutputName being the name of the operation that created the output. Might be confusing.